airbyte-webapp/src/views/Connection/ConnectionForm/components/ScheduleField.tsx

```diff
@@ -0,0 +1,182 @@
+
+    // Show cron and timezone fields based on the frequency
+    if (item.value === ConnectionScheduleType.cron) {
+      setIsCustomSchedule(true);
+    } else {
+      setIsCustomSchedule(false);
+    }
+  };
+
+  const getBasicScheduleValue = (value: ConnectionScheduleDataBasicSchedule) => {
+    // To set the initial value for the frequency dropdown. Only for Manual and Custom
+    if (!value) {
+      if (scheduleType === ConnectionScheduleType.cron) {
+        return formatMessage({
+          id: "frequency.cron",
+        }).toLowerCase();
+      }
+
+      return formatMessage({
+        id: "frequency.manual",
+      }).toLowerCase();
+    }
+
+    return value;
+  };
+
+  const getZoneValue = (currentSelectedZone: string) => {
+    if (!currentSelectedZone) {
+      return cronTimeZones[0].value;
+    }
+
+    return currentSelectedZone;
+  };
+
+  const onCronInputChange = (
+    event: ChangeEvent<HTMLInputElement>,
+    field: FieldInputProps<ConnectionScheduleDataCron>
+  ) => {
+    setFieldValue(field.name, {
+      ...field.value,
+      cronExpression: event.currentTarget.value,
+    });
+  };
+
+  const onCronZoneChange = (event: DropDownRow.IDataItem, field: FieldInputProps<ConnectionScheduleDataCron>) => {
+    setFieldValue(field.name, {
+      ...field.value,
+      cronTimeZone: event.value,
+    });
+  };
+
+  const cronTimeZones = useMemo(() => {
+    return availableCronTimeZones.map((zone: string) => ({ label: zone, value: zone }));
+  }, []);
+
+  return (
+    <>
+      <Field name="scheduleData.basicSchedule">
+        {({ field, meta }: FieldProps<ConnectionScheduleDataBasicSchedule>) => (
+          <div className={styles.flexRow}>
+            <div className={styles.leftFieldCol}>
+              <ControlLabels
+                className={styles.connectorLabel}
+                nextLine
+                error={!!meta.error && meta.touched}
+                label={formatMessage({
+                  id: "form.frequency",
+                })}
+                message={formatMessage({
+                  id: "form.frequency.message",
+                })}
+              />
+            </div>
+            <div className={styles.rightFieldCol} style={{ pointerEvents: mode === "readonly" ? "none" : "auto" }}>
+              <DropDown
+                {...field}
+                error={!!meta.error && meta.touched}
+                options={frequencies}
+                onChange={(item) => {
+                  onScheduleChange(item, field);
+                }}
+                value={getBasicScheduleValue(field.value)}
+              />
+            </div>
+          </div>
+        )}
+      </Field>
+      {isCustomSchedule && (
+        <Field name="scheduleData.cron">
+          {({ field, meta }: FieldProps<ConnectionScheduleDataCron>) => (
+            <div className={styles.flexRow}>
+              <div className={styles.leftFieldCol}>
+                <ControlLabels
```

Comment:
It might be helpful to add an `InfoTooltip` after this label with a link to some documentation about cron expressions, since some users may not be totally familiar with them. And they also seem to come in a few different formats, so it would be helpful to explain that and indicate which format we accept.

Reply:
Sure any suggestions on what we can add in the tooltip. I actually tried looking for a page in https://docs.airbyte.io but didn't find any 😢

Reply:
This is the page I looked at when I was testing your changes: https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm
so linking to that could be helpful, though I'm not sure if that is strange or not to link to an oracle page rather than our own docs.

@edmundito or @Amruta-Ranade do you have any thoughts here? Do you think it is important for us to link to documentation about cron expressions, or should we just expect that users who select `Cron` for their scheduling should know how to write them?